files/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: username
+  password: you_password
+  url: <%= ENV["DATABASE_URL"] %>
```

Comment:
@MProG we don't need `url` option here. It's needed only for `production` in order to have access to db via url and keep this url as env variable